xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-tree/xwiki-platform-index-tree-macro/src/main/resources/XWiki/DocumentTreeMacros.xml

```diff
@@ -787,7 +787,7 @@
 #macro (addPaginationNode $parentId $offset $totalCount $siblings)
   #set ($discard = $siblings.add({
     'id': "pagination:$parentId",
-    'text': "$!mathtool.sub($totalCount, $offset) more ...",
+    'text': "$!mathtool.sub($totalCount, $offset) $services.localization.render('index.documentTree.more')",
     'icon': 'fa fa-eye',
```

Comment:
See here for parametrized messages http://extensions.xwiki.org/xwiki/bin/view/Extension/Localization+Scripting .